app/services/widget-settings/chatbot.ts

```diff
@@ -0,0 +1,34 @@
+import { IWidgetData, IWidgetSettings, WidgetSettingsService } from './widget-settings';
+import { WidgetType } from 'services/widgets';
+import { clone } from 'lodash';
+import { $t } from 'services/i18n';
+
+export interface IMediaShareData extends IWidgetData {
+}
+
+export class ChatbotService extends WidgetSettingsService<IMediaShareData> {
```

Comment:
Since ChatBot is not only widget I suggest to rename `ChatbotService` -> `ChatBotWidgetService`. Check that this widget window opens without errors in offline mode.